sips/sip-111.md

```diff
@@ -0,0 +1,76 @@
+---
+sip: 111
+title: Enable Spartan Council voting on Synthetix Improvement Proposals (SIPs)
+status: Proposed
+author: Andy T CF (@andytcf), Kaleb Keny (@kaleb-keny), BigPenny (@BigPenny)
+discussions-to: https://research.synthetix.io/t/enable-spartan-council-voting-on-synthetix-improvement-proposals-sips/296
+created: 2021-02-07
+---
+
+<!--You can leave these HTML comments in your merged SIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new SIPs. Note that a SIP number will be assigned by an editor. When opening a pull request to submit your SIP, please use an abbreviated title in the filename, `sip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Simply describe the outcome the proposed changes intend to achieve. This should be non-technical and accessible to a casual community member.-->
+
+This SIP proposes to transition the Synthetix Improvement Proposal (SIP) voting process from discord #governance-polls to the current Spartan Council voting process.
+
+## Abstract
+
+<!--A short (~200 word) description of the proposed change, the abstract should clearly describe the proposed change. This is what *will* be done if the SIP is implemented, not *why* it should be done or *how* it will be done. If the SIP proposes deploying a new contract, write, "we propose to deploy a new contract that will do x".-->
+
+This proposal will enable proposed SIPs to be voted on by the Spartan Council. This SIP will transition the voting process from the #governance-polls channel to https://gov.synthetix.io. Voting power will not be removed from the Synthetix community but simply delegated to those who hold a Spartan Council NFT through the official Spartan Council Governance process, identical to how SCCP voting was delegated to the Spartan Council already.
```

Comment:
Voting power is delegated to the Spartan Council from tokens holders who stake their SNX via SIP-104. Council members hold a Spartan Council NFT and use this to vote for proposals through the official Spartan Council Governance process, identical to how SCCP voting was previously delegated to the Spartan Council.